Libplanet/Net/Swarm.cs

```diff
@@ -191,12 +191,16 @@ static Swarm()
 
         public Address Address => _privateKey.PublicKey.ToAddress();
 
-        public Peer AsPeer =>
-            new Peer(
+        public Peer AsPeer => EndPoint is null
+            ? new UnboundPeer(
+                _privateKey.PublicKey,
+                _appProtocolVersion,
+                _publicIPAddress) as Peer
```

Comment:
Is this casting(`as Peer`) necessary?

Reply:
Without this, ternary operator causes error.